api_v3/lib/types/entry/KalturaLiveEntry.php

```diff
@@ -274,4 +298,32 @@ protected function validateRecordingOptionsChanged($sourceObject, $attr)
 			$this->validateRecordingDone($sourceObject, "recordingOptions");
 		}
 	}
+
+	protected function validateSegmentDurationChanged($sourceObject, $attr)
```

Comment:
@lilachmaliniak I don't see this function is ever used, if this is the case please remove it